isubit/suitcase_interim
# LUGG-968 Added fontawesome to suitcase_interim

This pr replaces the ISU cdn's version of font_awesome with our own assets.

To test:
- Pull down this branch
- Clear cache/rebuild registry a bazillion times
- Use the social menu to add a link to Facebook and/or Twitter
- Does the respective icon appear?
